apps/web/src/server/views/applications/components/case-form/case-form-layout.njk

```diff
@@ -73,12 +74,12 @@
 {% macro renderComponent(componentName, layout) %}
 	{% if componentName == 'title' %}
 		{{ title('title', values, errors) }}
-  {% elif componentName == 'titleWelsh' %}
-    {{ title('titleWelsh', values, errors) }}
+	{% elif componentName == 'titleWelsh' %}
+		{{ title('titleWelsh', values, errors) }}
 	{% elif componentName == 'description' %}
 		{{ description('description', values, errors) }}
-  {% elif componentName == 'descriptionWelsh' %}
-    {{ description('descriptionWelsh', values, errors) }}
+	{% elif componentName == 'descriptionWelsh' %}
+		{{ description('descriptionWelsh', values, errors) }}
```

Comment:
There's some parts of the diff that are just changing tabs to spaces. Not a big deal but worth seeing if your text editor can detect when tabs are used and stay consistent with what's already there :)